.repo/sample-list-generator/src/constants.ts

```diff
@@ -0,0 +1,16 @@
+export type FolderTypes = "API_SAMPLE" | "FUNCTIONAL_SAMPLE";
+
+// Define all available folders for samples
+export const AVAILABLE_FOLDERS = [
+  {
+    path: 'api-samples',
+    type: 'API_SAMPLE'
```

Comment:
nit: Why not use the type defined above?

Reply:
The assertion here is unnecessary. Could I change it to the following code?

```ts
export const AVAILABLE_FOLDERS: { path: string, type: FolderTypes }[] =
```

Reply:
Removing the assertion seems good. I don't think it's possible to use the type directly because it's just a string union.

Reply:
Done. https://github.com/GoogleChrome/chrome-extensions-samples/pull/963/commits/bd56b470d789136a3260124b355e19ef22ae586c